app/Jobs/MigrationWikiCreate.php

```diff
@@ -111,6 +112,11 @@ private function createWiki($user, $wikiDetails, $wikiDb) {
             ]);
         });
 
+        // Create k8s ingress if the wiki uses a custom domain
+        if ( isset($wiki->domain) && ! WikiController::isSubDomain($wiki->domain)) {
```

Comment:
Cool, this first isset to me looks like a more serious failure that might warrant marking the job as a failure if that happens. If we get down here and the $wiki object is incomplete that probably means the transaction failed for some reason but the migration job would still be considered valid? I guess ultimately this depends on how we run these things. If we don't see it happening in some controller at some point but rather from the command line every time, maybe this doesn't matter?